bundles/org.openhab.binding.digitalstrom/src/main/resources/OH-INF/i18n/digitalstrom_de.properties

```diff
@@ -54,7 +54,7 @@ gr_label = GR-Klemme (grau)
 gr_desc =  Repräsentiert eine Schatten-Klemme.
 
 bl_label = BL-Klemme (blau)
-bl_desc = Repräsentiert eine Heizungs-Klemme.
+bl_desc =  Repräsentiert eine Heizungs-Klemme.
```

Comment:
I see unexpected leading space added to several entries.

Reply:
Seems correct at Crowdin:
https://crowdin.com/translate/openhab-addons/2095/en-de?filter=basic&value=3#42121

Maybe a problem with the script for exporting translations? Even previewing the original source has this formatting:
![image](https://user-images.githubusercontent.com/19519842/160289261-428eee8d-6125-434f-a85e-29db15a96fc7.png)

Reply:
@cweitkamp @wborn you are generally our specialists with Crowdin.

Reply:
It needs to be removed from the default translations file. Otherwise Crowdin will complain to translators that the space is missing when translations are made, see:

https://github.com/openhab/openhab-addons/blob/c22a33d651bb382e21cbdc5c2ae39c4dc5029636/bundles/org.openhab.binding.digitalstrom/src/main/resources/OH-INF/i18n/digitalstrom.properties#L57

Reply:
Ah ok, I did not understand that the problem was in the default translations file.

Reply:
Doing a grep, I found leading spaces for 7 bindings:
```
org.openhab.binding.amazondashbutton/src/main/resources/OH-INF/i18n/amazondashbutton.properties:dashButtonPressChannelDescription =  Channel for recognizing presses on the Amazon Dash Button
org.openhab.binding.amazondashbutton/src/main/resources/OH-INF/i18n/amazondashbutton_de.properties:dashButtonPressChannelDescription =  Channel um eine Betätigung des Amazon Dash Button festzustellen.
org.openhab.binding.amazondashbutton/src/main/resources/OH-INF/i18n/amazondashbutton_hu.properties:dashButtonPressChannelDescription =  Felismerési csatorna az Amazon Dash gomb megnyomásához
org.openhab.binding.amazondashbutton/src/main/resources/OH-INF/i18n/amazondashbutton_it.properties:dashButtonPressChannelDescription =  Canale per riconoscere la pressione di un pulsante Amazon Dash
org.openhab.binding.amazondashbutton/src/main/resources/OH-INF/i18n/amazondashbutton_uk.properties:dashButtonPressChannelDescription =  Канал розпізнавання натискань кнопки Amazon Dash
org.openhab.binding.bmwconnecteddrive/src/main/resources/OH-INF/i18n/bmwconnecteddrive_de.properties:channel-type.bmwconnecteddrive.range-radius-fuel-channel.label =  Verbrenner Reichweiten-Radius
org.openhab.binding.digitalstrom/src/main/resources/OH-INF/i18n/digitalstrom.properties:ge_desc =  Represents a digitalSTROM light device.
org.openhab.binding.digitalstrom/src/main/resources/OH-INF/i18n/digitalstrom.properties:sw_desc =  Represents a digitalSTROM joker device.
org.openhab.binding.digitalstrom/src/main/resources/OH-INF/i18n/digitalstrom.properties:gr_desc =  Represents a digitalSTROM shade device.
org.openhab.binding.digitalstrom/src/main/resources/OH-INF/i18n/digitalstrom.properties:bl_desc =  Represents a digitalSTROM heating device.
org.openhab.binding.digitalstrom/src/main/resources/OH-INF/i18n/digitalstrom.properties:dsisens200_desc =  Represents the digitalSTROM sensor device dS-iSens200 for indoor temperature and relative humidity.
org.openhab.binding.digitalstrom/src/main/resources/OH-INF/i18n/digitalstrom.properties:electric_meter_label =  Electric meter
org.openhab.binding.digitalstrom/src/main/resources/OH-INF/i18n/digitalstrom.properties:binary_input_wind_strenght_above_limit_desc =  Will be activated, if wind strength is above a user adjusted limit.
org.openhab.binding.digitalstrom/src/main/resources/OH-INF/i18n/digitalstrom_de.properties:ge_desc =  Repräsentiert eine Licht-Klemme.
org.openhab.binding.digitalstrom/src/main/resources/OH-INF/i18n/digitalstrom_de.properties:sw_desc =  Repräsentiert eine Joker-Klemme.
org.openhab.binding.digitalstrom/src/main/resources/OH-INF/i18n/digitalstrom_de.properties:gr_desc =  Repräsentiert eine Schatten-Klemme.
org.openhab.binding.digitalstrom/src/main/resources/OH-INF/i18n/digitalstrom_de.properties:dsisens200_desc =  Repräsentiert das digitalSTROM Sensor-Gerät dS-iSens200 zur Messung der Temperatur und der relativen Luftfeuchtigkeit eines Raums.
org.openhab.binding.modbus.helioseasycontrols/src/main/resources/OH-INF/i18n/helioseasycontrols_de.properties:thing-type.modbus.helios-easycontrols.group.co2Control.label =  CO2-Steuerung
org.openhab.binding.modbus.helioseasycontrols/src/main/resources/OH-INF/i18n/helioseasycontrols_de.properties:channel-type.modbus.minFanStage.description =  Minimale Lüfterstufe (0 oder 1)
org.openhab.binding.netatmo/src/main/resources/OH-INF/i18n/netatmo_fr.properties:channel-type.netatmo.pressureTrend.state.option.up =  A la hausse
org.openhab.binding.onewire/src/main/resources/sensor.properties:DS2406.channels =  digital0:dio:Digital I/O 0,digital1:dio:Digital I/O 1
org.openhab.binding.onewire/src/main/resources/sensor.properties:DS2408.channels =  digital0:dio:Digital I/O 0,digital1:dio:Digital I/O 1,digital2:dio:Digital I/O 2,digital3:dio:Digital I/O 3,digital4:dio:Digital I/O 4,digital5:dio:Digital I/O 5,digital6:dio:Digital I/O 6,digital7:dio:Digital I/O 7
org.openhab.binding.onewire/src/main/resources/sensor.properties:DS2413.channels =  digital0:dio:Digital I/O 0,digital1:dio:Digital I/O 1
org.openhab.binding.pixometer/src/main/resources/OH-INF/i18n/pixometer.properties:binding.pixometer.description =  This binding can connect to the Pixometer.io API and accesses meters and measurement data from the connected account.
org.openhab.binding.pixometer/src/main/resources/OH-INF/i18n/pixometer.properties:channel-type.pixometer.last_reading_date.description =  Date of the last reading that has been made with pixometer.
org.openhab.binding.pixometer/src/main/resources/OH-INF/i18n/pixometer_de.properties:binding.pixometer.description =  Mit dem Pixometer Binding ist es möglich über die Pixometer.io API Daten und Messwerte von den dort erfassten Zählern abzurufen.
org.openhab.binding.pixometer/src/main/resources/OH-INF/i18n/pixometer_de.properties:channel-type.pixometer.last_reading_date.description =  Der Zeitpunkt der zuletzt abgefragten Ablesung für diesem Zähler.
```

Reply:
@lolodomo - so I'd propose merging this PR as it's consistent with the default translation file. Another PR can be made for trimming the defaults which will then propagate to Crowdin and back.

Reply:
I agree with you.